### DISCUSSION TITLE
Does BleskOS support multiple cores ?

### DISCUSSION THREAD

**Discussion by lotusdew1980** [VendelinSlezak/BleskOS#82] [Q&A] [answered]
Hi, this is a great OS to learn from. Complex enough to do interesting things but also simple enough to understand. Thanks a lot for your work. I had a question - which dosent seem to pop out. Does it support multicore processors ?

**Comment by VendelinSlezak** [comment, marked as answer]
Hello, I am glad you like this project :)

Actual version of BleskOS do not use multiple cores on processor. You can think of it as one big program that runs on one core. So it will run on any processor regardless of number of cores, but it will not use it's full potential.
